make/autoconf/platform.m4

```diff
@@ -581,6 +581,8 @@ AC_DEFUN([PLATFORM_SETUP_LEGACY_VARS_HELPER],
     HOTSPOT_$1_CPU_DEFINE=S390
   elif test "x$OPENJDK_$1_CPU" = xs390x; then
     HOTSPOT_$1_CPU_DEFINE=S390
+  elif test "x$OPENJDK_$1_CPU" = xloongarch64; then
+    HOTSPOT_$1_CPU_DEFINE=LOONGARCH64
```

Comment:
Would you mind moving the preceding two lines for RISCV32 to after PPC32? https://github.com/openjdk/jdk/blob/master/make/autoconf/platform.m4#L570C1-L571C34
We only have Zero support for RISCV32 for now. Thanks.